### PR TITLE
Added loading snapshot flow.

### PR DESCRIPTION
**Pull request by ndemidovich-sc**
Behavior:
- Possibility to download snapshot is visible in case if progress less than 50%
- The peer network stops in time until the snapshot workflow is completed

Snapshot workflow steps:
- Downloaing
- Clearing DB
- Unarchiving
- Clearing temp files

Problems:
In current version of SDK if peer network is stopped, then it cannot return balances by accounts. SDK throw error in this case.
```
RangeError [ERR_OUT_OF_RANGE]: The value of "value" is out of range. It must be >= 0 and <= 4294967295. Received -1
    at new NodeError (node:internal/errors:371:5)
    at checkInt (node:internal/buffer:72:11)
    at Buffer.writeUInt32BE (node:internal/buffer:815:10)
    at U32EncodingBE.serialize (wallet-app\node_modules\@ironfish\sdk\build\src\storage\database\encoding.js:41:16)
    at WalletDB.loadUnspentNoteValues (wallet-app\node_modules\@ironfish\sdk\build\src\wallet\walletdb\walletdb.js:287:112)  
    at loadUnspentNoteValues.next (<anonymous>)
    at Account.calculateAvailableBalance (wallet-app\node_modules\@ironfish\sdk\build\src\wallet\account.js:576:26) {        
  code: 'ERR_OUT_OF_RANGE'
}
```

### CODE REVIEW

**Comment by ndemidovich-sc** [comment]
@skylarrichard123 You can use this demo as base point for Snapshot UI.

**Comment by skylarrichard123** [comment]
We will detect whether or not the snapshot should be downloaded automatically, so the user won't have a choice in the flow. The reason being is this app is meant for people with lower tech savvy, so that might be a point of confusion for a lot of users.

**Review by azhurauski-sc — COMMENTED** [review]
some suggestions

**Review by skylarrichard123 — CHANGES_REQUESTED** [review]
Change requests here:
https://www.figma.com/file/fYw3aNgfS0qCS2jezGqv8P/Wallet-(New-Ui-Kit)?node-id=5000%3A21269&t=gZpgFbCkitM07Q7s-1

File: src/components/Snapshot/SnapshotStatusModal.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+              <Box>
+                <chakra.h5 color={NAMED_COLORS.GREY}>
+                  {status?.estimate
+                    ? formatRemainingTime(status?.estimate)
+                    : '~'}
+                </chakra.h5>
+              </Box>
+            </Flex>
+            <Box w="100%">
+              <Progress
```
Comment (azhurauski-sc) [inline src/components/Snapshot/SnapshotStatusModal.tsx:124]: there are two variants available `ironBlue` and `ironLightBlue` for Progress

**Pull request merged**